src/providers/Anonymous.ts

```diff
@@ -66,13 +72,13 @@ export function Anonymous<DataModel extends GenericDataModel>(
     id: "anonymous",
     authorize: async (params, ctx) => {
       const profile = config.profile?.(params, ctx) ?? { isAnonymous: true };
-      const { user } = await createAccount(ctx, {
+      const { account } = await createAccount(ctx, {
```

Comment:
one concern is that folks who have made their own providers may be tripped up by `createAccount` not returning the user anymore. This would be a breaking change technically